kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ListPartitionOffsetsActionIntegrationTest.java

```diff
@@ -0,0 +1,172 @@
+
+package io.confluent.kafkarest.integration.v3;
+
+import static io.confluent.kafkarest.TestUtils.TEST_WITH_PARAMETERIZED_QUORUM_NAME;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.common.collect.ImmutableMap;
+import io.confluent.kafkarest.entities.v3.ListPartitionOffsetsResponse;
+import io.confluent.kafkarest.entities.v3.PartitionWithOffsetsData;
+import io.confluent.kafkarest.entities.v3.Resource;
+import io.confluent.kafkarest.integration.ClusterTestHarness;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class ListPartitionOffsetsActionIntegrationTest extends ClusterTestHarness {
+
+  private static final String TOPIC_NAME = "topic-1";
+
+  private static final int PARTITION_ID = 0;
+
+  private static final String OFFSET_TYPE = "earliest_and_latest";
+
+  private String expectedOffsetsUrl;
+
+  public ListPartitionOffsetsActionIntegrationTest() {
+    super(/* numBrokers= */ 1, /* withSchemaRegistry= */ false);
+  }
+
+  @BeforeEach
+  @Override
+  public void setUp(TestInfo testInfo) throws Exception {
+    super.setUp(testInfo);
+    createTopic(TOPIC_NAME, 1, (short) 1);
+    expectedOffsetsUrl =
+        "/v3/clusters/"
+            + getClusterId()
+            + "/topics/"
+            + TOPIC_NAME
+            + "/partitions/"
+            + PARTITION_ID
+            + "/offset";
+  }
+
+  @ParameterizedTest(name = TEST_WITH_PARAMETERIZED_QUORUM_NAME)
+  @ValueSource(strings = {"kraft"})
+  public void listPartitionOffsets_existingPartitionWithOffsets_returnsPartitionWithOffsets(
+      String quorum) throws Exception {
+    String baseUrl = restConnect;
+    String clusterId = getClusterId();
+
+    ListPartitionOffsetsResponse expected =
+        ListPartitionOffsetsResponse.create(
+            PartitionWithOffsetsData.builder()
+                .setMetadata(
+                    Resource.Metadata.builder()
+                        .setSelf(
+                            baseUrl
+                                + "/v3/clusters/"
+                                + clusterId
+                                + "/topics/"
+                                + TOPIC_NAME
+                                + "/partitions/"
+                                + PARTITION_ID
+                                + "/offset")
+                        .setResourceName(
+                            "crn:///kafka="
+                                + clusterId
+                                + "/topic="
+                                + TOPIC_NAME
+                                + "/partition="
+                                + String.valueOf(PARTITION_ID)
+                                + "/offset_type="
+                                + OFFSET_TYPE)
```

Comment:
can you include a test for default behaviour? i.e. no `offset_type` specified is same as `earliest_and_offset`.